ndsl/__init__.py

```diff
@@ -1 +1,62 @@
+from .buffer import Buffer
+from .comm.boundary import Boundary, SimpleBoundary
+from .comm.communicator import CubedSphereCommunicator, TileCommunicator
+from .comm.local_comm import AsyncResult, ConcurrencyError, LocalComm
+from .comm.mpi import MPIComm
+from .comm.null_comm import NullAsyncResult, NullComm
+from .comm.partitioner import CubedSpherePartitioner, TilePartitioner
+from .constants import ConstantVersions
+from .dsl.caches.codepath import FV3CodePath
+from .dsl.dace.dace_config import DaceConfig, DaCeOrchestration, FrozenCompiledSDFG
+from .dsl.dace.orchestration import orchestrate, orchestrate_function
+from .dsl.dace.utils import (
+    ArrayReport,
+    DaCeProgress,
+    MaxBandwithBenchmarkProgram,
+    StorageReport,
+)
+from .dsl.dace.wrapped_halo_exchange import WrappedHaloUpdater
+from .dsl.stencil import (
+    CompareToNumpyStencil,
```

Comment:
Still don't think CompareToNumpyStencil needs to be here

Reply:
We can bury it, down to stencil